### ISSUE TITLE
Broken scrolling in iOS 11 

### ISSUE THREAD

**Issue by 3lvis** [bakkenbaeck/SectionScrubber#52]
Before the scrubber started just after the navigation bar ended, not it doesn't do that anymore.

**Comment by askeland** [comment]
Probably because of the new safe area stuff and the iPhone X nudge:

```
if #available(iOS 11.0, *) {
    self.collectionView.contentInsetAdjustmentBehavior = .never
}
```

**Comment by 3lvis** [comment]
That didn't work =/
